layouts/explore-layout.vue

```diff
@@ -15,14 +19,14 @@
         error-subtitle="Please check your network connections"
         error-title="Offline Detected" />
       <div v-else>
-        <section class="section">
+        <section class="section py-2">
           <div class="container">
             <h1 v-if="isExplore" class="title">{{ $t('explore') }}</h1>
 
             <ExploreTabsFilterSort />
           </div>
         </section>
-        <hr class="m-0" />
+        <hr class="text-color" />
```

Comment:
Now we have a whitespace if you remove this

![Screenshot 2023-03-06 at 09-13-40 Denver waifus 🤎 KodaDot - NFT Market Explorer](https://user-images.githubusercontent.com/9987732/223054062-d2df955b-c195-41ff-9b05-c893d22bf466.png)

Reply:
yep yep, missed that.
Thank you